lib/src/core1/voronoiMaskMerger.hpp

```diff
@@ -14,7 +14,7 @@ namespace Core {
 class VoronoiMaskMerger : public MaskMerger {
  public:
   VoronoiMaskMerger() : feather(0) {}
-  ~VoronoiMaskMerger() {}
+  virtual ~VoronoiMaskMerger() override {}
```

Comment:
There is a bit a discussion on stackoverflow, whether the override keyword is needed for virutal destructors (because they are not overriding anything, they are chained). 
The virtual keyword may also be redundant, since it derived from a class with a virtual destructor, the destructor in the derived class will implicitely be virtual.

Reply:
I've had a quick look too, yikes, another controversial topic in C++. Indeed, it's redundant when the base class has a virtual destructor. But it seems to be a good safeguard about accidental removal of the virtual keyword on the base class destructor. Then it wouldn't compile any more (https://stackoverflow.com/a/17923488/463796). So I think we can leave it in.